checks/apps/pytorch/nvcr_utility.py

```diff
@@ -0,0 +1,28 @@
+
+import requests
+import re
+from packaging.version import Version
+
+
+def latest_nvidia_image_tags(image_name:str):
+
+    token_response = requests.get(f"https://nvcr.io/proxy_auth?scope=repository:nvidia/pytorch:pull")
```

Comment:
Is it possible to use the image name here?

Reply:
not only possible, it must!
I was doing some debugging and forgot to change it back